packages/client/src/v2-events/features/events/components/Review.tsx

```diff
@@ -329,96 +334,102 @@ function ReviewComponent({
           </HeaderContainer>
           <FormData>
             <ReviewContainter>
-              {formConfig.pages.map((page) => {
-                return (
-                  <DeclarationDataContainer
-                    key={'Section_' + page.title.defaultMessage}
-                  >
-                    <Accordion
-                      action={
-                        <Link
-                          onClick={(e) => {
-                            e.stopPropagation()
-                            onEdit({ pageId: page.id })
-                          }}
-                        >
-                          {intl.formatMessage(reviewMessages.changeButton)}
-                        </Link>
-                      }
-                      expand={true}
-                      label={intl.formatMessage(page.title)}
-                      labelForHideAction="Hide"
-                      labelForShowAction="Show"
-                      name={'Accordion_' + page.id}
+              {formConfig.pages
+                .filter(({ hideOnReview }) => !hideOnReview)
```

Comment:
Could hiding a page be based on if any of the fields on the page pass `isFieldVisible`? Wouldn't need a config flag